methods/lcarun.ts

```diff
@@ -2,28 +2,102 @@ import knex from 'knex';
 
 import { Lci } from 'models/schema';
 import { RunParams } from './lca.model';
-// import { listenerCount } from 'cluster';
+
+// NonrenewableUnitParams were used to change units for nonrenewable group. Units change is not needed for other groups
+interface NonrenewableUnitParams {
+  'brownCoal': number;
+  'hardCoal': number;
+  'crudeOil': number;
+  'mineGas': number;
+  'naturalGas': number;
+  'Uranium': number;
+  [key: string]: number;
+}
+const unitParams: NonrenewableUnitParams = { 'brownCoal': 9.9, 'hardCoal': 19.1, 'crudeOil': 45.8,
+                                             'mineGas': 32.45, 'naturalGas': 32.12, 'Uranium': 560000};
 
 const lcarun = async (params: RunParams, db: knex) => {
-  const rows: Lci[] = await db
+  const inputRows: Lci[] = await db
     .table('lci_input')
-    .where({ lci_group: 'renewable' });
+    .where({ lci_group: 'renewable' }).orWhere({ lci_group: 'nonrenewable' }).orWhere({ lci_group: 'water' });
+
+  const outputRows: Lci[] = await db
+    .table('lci_output')
+    .where({ lci_group: 'CO2' }).orWhere({ lci_group: 'CH4' }).orWhere({ lci_group: 'N2O' })
+    .orWhere({ lci_group: 'CO' }).orWhere({ lci_group: 'NOx' }).orWhere({ lci_group: 'NMVOC' })
+    .orWhere({ lci_group: 'particulates' });
+  // const nonrenewableRows: Lci[] = await db
+  //   .table('lci_input')
+  //   .where({ lci_group: 'nonrenewable' });
+  // const waterRows: Lci[] = await db
+  //   .table('lci_input')
+  //   .where({ lci_group: 'water' });
 
   // const bigNum = Math.max(...rows.map(r => r.number));
-  let sum = 0;
-  for (let i = 0; i < rows.length; i++) {
-      sum = sum + processRow(rows[i], params);
+  let renewableSum = 0;
+  let nonrenewableSum = 0;
+  let waterSum = 0;
+// tslint:disable-next-line: variable-name
+  let CO2sum = 0;
+// tslint:disable-next-line: variable-name
+  let CH4sum = 0;
+// tslint:disable-next-line: variable-name
+  let N2Osum = 0;
+// tslint:disable-next-line: variable-name
+  let COsum = 0;
+// tslint:disable-next-line: variable-name
+  let NOxsum = 0;
+// tslint:disable-next-line: variable-name
+  let NMVOCsum = 0;
+  let particulatesSum = 0;
+// tslint:disable-next-line: variable-name
+  let CO2eSum = 0;
+  for (let i = 0; i < inputRows.length; i++) {
```

Comment:
I would break these into two different functions

Reply:
did that in the new branch, KL/lci_calculation.